### PR TITLE
Add test for command-not-found tool

### PR DESCRIPTION
**Pull request by asdil12**
To catch regressions for http://bugzilla.suse.com/show_bug.cgi?id=952496

https://progress.opensuse.org/issues/9684

### CODE REVIEW

**Comment by coolo** [comment]
Can you save_screenshot; so we can review the results? 

But I don't think this test should run in every test run - put it where a2ps.pm is.

**Comment by coolo** [comment]
and IMO: make it important, otherwise it's pointless to test. Noone reviews unimportant test failures

**Comment by asdil12** [comment]
It acually already created a screenshot implicitly:
http://artemis.suse.de/tests/546/modules/command_not_found/steps/1

File: tests/console/command_not_found.pm [hunk 1]
```diff
@@ -0,0 +1,23 @@
+# SUSE's openQA tests
+#
+# Copyright © 2009-2013 Bernhard M. Wiedemann
+# Copyright © 2012-2016 SUSE LLC
```
Comment (aaannz) [inline tests/console/command_not_found.pm]: Copy paste ©. Only 2016 is valid.

Comment (asdil12) [inline tests/console/command_not_found.pm]: I don't know, what you mean with this comment.

**Comment by asdil12** [comment]
@coolo
Marked test as important and put it into extra_tests.
For the screenshot thing see my comment above.

**Comment by asdil12** [comment]
I had a look at the other tests and as `save_screenshot` after a `script`run` seems to be common, I added a `save`screenshot` to my patch as well.

File: tests/console/command_not_found.pm (hunk 1 above)
Comment (coolo) [inline tests/console/command_not_found.pm]: the proper copyright is 

Copyright © 2016 SUSE LLC

**Comment by coolo** [comment]
assert_script_run does a screenshot as it does wait_serial, so yes - the screenshot is not really adding anything. But leave it in, as you noticed there are many other tests doing that.

**Pull request merged**